api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/MessageProcessorScheduler.java

```diff
@@ -20,6 +20,7 @@
 @Slf4j
 @RequiredArgsConstructor
 @Component
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
I think this is fine for a class like this, splitting this into multiple components just to keep PMD happy isn't the right thing to do.

Reply:
If the methods contained logic then we probably would want to split it up. As they're just scheduling tasks it makes sense to me to keep them together.